huaweicloud/config.go

```diff
@@ -618,6 +618,10 @@ func (c *Config) ElbV2Client(region string) (*golangsdk.ServiceClient, error) {
 	return c.NewServiceClient("elbv2", region)
 }
 
+func (c *Config) Elb_V2Client(region string) (*golangsdk.ServiceClient, error) {
```

Comment:
how about rename the client as `LoadBalancerClient`?
there are many clients for elb service, sugget to add some comments here

Reply:
fixed